tests/components/airgradient/conftest.py

```diff
@@ -0,0 +1,51 @@
+"""AirGradient tests configuration."""
+
+from collections.abc import Generator
+from unittest.mock import patch
+
+from airgradient import Measures
+import pytest
+
+from homeassistant.components.airgradient.const import DOMAIN
+from homeassistant.const import CONF_HOST
+
+from tests.common import MockConfigEntry, load_fixture
+from tests.components.smhi.common import AsyncMock
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.airgradient.async_setup_entry",
```

Comment:
This is weird as it means you are not actually doing anything ?

Reply:
Only for the config flow tests. In config flow tests we only test the config flow, not the setup of the integration. This code is also present in the scaffold so it's not something strange for the tests to have